scripts/upgrade.py

```diff
@@ -0,0 +1,141 @@
+import yaml
+
+
+if __name__ == "__main__":
+    with open("helm/values.yaml", "r") as f:
+        config = yaml.safe_load(f)
+
+    ### Create backup copy
+    with open("helm/values.yaml.old", "w") as f:
+        yaml.dump(config, f)
+
+    ### Remove image tag
+    image = config["deployment"]["image"]
+    repo, _ = image.split(":")
```

Comment:
```suggestion
    repo = "".join(image.split(":")[:-1])
```
just in case someone pushed the connaisseur image into the own private registry at `private-registry:3000/library/connaisseur:v2`